iml-api/src/graphql.rs

```diff
@@ -659,6 +659,8 @@ impl MutationRoot {
                     interval
                 )
                 VALUES ($1, $2, $3)
+                ON CONFLICT (filesystem_name, interval)
+                DO UPDATE SET use_barrier = EXCLUDED.use_barrier
```

Comment:
We should ```DO NOTHING``` here. If we change a param, the backing timer will need to be regenerated.